amr-wind/utilities/sampling/LidarSampler.cpp

```diff
@@ -0,0 +1,91 @@
+#include "amr-wind/utilities/sampling/LidarSampler.H"
+#include "amr-wind/CFDSim.H"
+#include "amr-wind/utilities/tensor_ops.H"
+#include "amr-wind/utilities/linear_interpolation.H"
+
+#include "AMReX_ParmParse.H"
+
+namespace amr_wind {
+namespace sampling {
+
+LidarSampler::LidarSampler(const CFDSim& sim) : LineSampler(sim) {}
+
+void LidarSampler::initialize(const std::string& key)
+{
+    // Read in new inputs specific to this class
+    amrex::ParmParse pp(key);
+
+    // This is the origin of the scan (x, y, z) [m]
+    pp.getarr("origin", m_origin);
+
+    // The number of points
+    pp.get("num_points", m_npts);
+
+    // The length of the beam [m]
+    pp.get("length", m_length);
+
+    // The time table [s]
+    pp.getarr("time_table", m_time_table);
+    // Azimuth angle (this is in the N, E, S, W direction) [degrees]
```

Comment:
Is this compass direction values or angle with reference to the inertial reference frame axes?

Reply:
Yes, this is an inertial frame. I specified the orientation in the documentation.